config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  namespace :api do
+    get 'orders/create'
+  end
```

Comment:
?? Why do you add the api namespace twice?

Reply:
We have no good answer to that really, but we have corrected it to one time now.